api/database/seeders/PoolTestSeeder.php

```diff
@@ -43,25 +43,196 @@ public function run()
                 'closing_date' => config('constants.far_future_date'),
                 'publishing_group' => PublishingGroup::IAP->name,
             ],
-        ];
+            
+            //  IT-01  - Draft Job
+            [
+                'name' => [
```

Comment:
Where did these names come from?  It might be nice if you used the names from the document.  Same with the users.